### ISSUE TITLE
Add "v2" and "2" tags to dockerhub releases 

### ISSUE THREAD

**Issue by Vinigas** [traefik/traefik-library-image#12]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.containo.us) and didn't find any.

### What did you expect to see?

I expected that `docker pull traefik:v2` will not result in not existing image.

I use traefik in docker swarm and watchtower to automatically update. I don't want to use latest tag, in case v3 is released.

**Comment by LorneCash** [comment]
I agree, can we please get this?

**Comment by dumbasPL** [comment]
`v3` and `3` as well please

**Comment by MortenCB** [comment]
This would be very helpful to keep it at latest version 2 and not upgrade to version 3 before I'm ready to do a lot of config changes.  Please add a latest version 2 tag, so I don't have to manually keep an eye on which is the latest and change the version manually.

**Comment by GeorchW** [comment]
Yes, please! We usually want to be on the latest version, but got a bit of downtime for all of our services after v3 was released. If we could just pin to the major version, that would be just about right...

**Comment by GeorchW** [comment]
@vini-nu could you rename the issue to `Add major version tags to dockerhub releases (e.g. "v2", "2", "v3", "3")` or similar?

**Comment by dumbasPL** [comment]
closed as completed without comment or commit? I'm not seeing the new tags.

**Comment by mmatur** [comment]
Hi @dumbasPL,

Sorry about that, we have an internal tool that generates Docker manifest file for official Docker build. This tool has been updated internally and for the next Traefik release the tags `v2`, `2`, `v3` and `3` will be available

This is why there is no commit associated

Linked to https://github.com/traefik/traefik/issues/11057
